hladny_matfyzak.py

```diff
@@ -0,0 +1,57 @@
+from brutal.core.plugin import cmd
+from datetime import datetime
+import hladnymatfyzak
+
+
+DATE_FORMAT = '%d.%m.%Y'
+
+
+def validate_date_and_args(args):
+    """Validates given date"""
+    if len(args) == 1:
```

Comment:
@Adman so even if we get more than one argument the result will be just `datetime.today`? That does not feel right.